packages/visx-responsive/test/withParentSize.test.tsx

```diff
@@ -20,7 +22,7 @@ describe('withParentSize', () => {
 
   test('it should pass parentWidth and parentHeight props to its child', () => {
     const HOC = withParentSize(Component, ResizeObserver);
-    const { getByTestId } = render(<HOC initialWidth={200} initialHeight={200} />);
+    const { getByTestId } = render(<HOC role="img" initialWidth={200} initialHeight={200} />);
```

Comment:
Could also think about an inverse test like this

```suggestion
    // @ts-expect-error ensure unknown types still error
    render(<HOC unknown="prop");
    
    const { getByTestId } = render(<HOC role="img" initialWidth={200} initialHeight={200} />);
```